BookEcommerce.Models/DAL/Repositories/CartRepository.cs

```diff
@@ -0,0 +1,23 @@
+﻿using BookEcommerce.Models.DAL.Interfaces;
+using BookEcommerce.Models.Entities;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookEcommerce.Models.DAL.Repositories
+{
+    public class CartRepository : Repository<Cart>, ICartRepository
+    {
+        public CartRepository(DbFactory dbFactory) : base(dbFactory)
+        {
+        }
+
+        public async Task<List<Cart>> GetCart(string id)
+        {
+            return await GetQuery(crt => crt.CustomerId.Contains(id)).ToListAsync();
```

Comment:
So Id is UserGuid or CartGuid. Btw `Contains` does not make any sense for getting with using Id